pkg/controller/backup_deletion_controller.go

```diff
@@ -442,6 +457,66 @@ func (c *backupDeletionController) deleteResticSnapshots(backup *v1.Backup) []er
 	return errs
 }
 
+func getCSIVolumeSnapshotsInBackup(b *velerov1api.Backup, csiClient snapshotter.SnapshotV1beta1Interface, log *logrus.Entry) ([]snapshotv1beta1api.VolumeSnapshot, []error) {
+	csiVolSnaps := []snapshotv1beta1api.VolumeSnapshot{}
+	errs := []error{}
+
+	selector := metav1.LabelSelector{
+		MatchLabels: map[string]string{
+			velerov1api.BackupNameLabel: label.GetValidName(b.Name),
+		},
+	}
+	listOpts := metav1.ListOptions{
+		LabelSelector: metav1.FormatLabelSelector(&selector),
+	}
+
+	for _, ns := range pkgbackup.GetNamespacesInBackup(b) {
+		log.Debugf("Listing CSI volumesnapshots in namespace %s", ns)
+		nsvs, err := csiClient.VolumeSnapshots(ns).List(listOpts)
+		if err != nil {
+			errs = append(errs, err)
+		}
+		if nsvs.Items != nil && len(nsvs.Items) != 0 {
+			log.Debugf("Found %d CSI volumesnapshots in namespace %s", len(nsvs.Items), ns)
+			csiVolSnaps = append(csiVolSnaps, nsvs.Items...)
+		}
+	}
+
+	log.Infof("Found %d CSI volumesnapshots in backup %s", len(csiVolSnaps), b.Name)
+	return csiVolSnaps, errs
+}
+
+func setVSCDeletionPolicyToDelete(vscName string, csiClient snapshotter.SnapshotV1beta1Interface) (*snapshotv1beta1api.VolumeSnapshotContent, error) {
```

Comment:
the value of this helper is dubious now that it's just 2 lines, I would consider dropping it and just inlining this code where it's used.